src/components/dialog/DialogDismissibleView.js

```diff
@@ -156,14 +156,12 @@ class DialogDismissibleView extends PureComponent {
   };
 
   animateTo = (toValue, animationEndCallback) => {
-    const animation = Animated.timing(this.animatedValue, {
+    Animated.timing(this.animatedValue, {
       toValue,
-      duration: 400,
-      easing: Easing.bezier(0.65, 0, 0.35, 1),
+      duration: 300,
+      easing: Easing.bezier(0.2, 0, 0.35, 1),
       useNativeDriver: true
-    });
-
-    this.setState({isAnimating: true}, () => animation.start(animationEndCallback));
```

Comment:
DialogDismissibleView is an internal component of dialog and being used only with isAnimated = true
Maybe it's indeed redundant.. should I remove it ?

Reply:
Not sure if it's a typo or not, I wrote `isAnimating` and not `isAnimated` as in `this. isAnimating`.
It is checked in:
```
if (hide && !this.isAnimating) {
  this.hide();
}
```
But since it is always false, it seems it can be removed.

Reply:
My previous comment was about something else.. I got confused. nvm..
Anyway, I will remove this.isAnimating